ui/appui-react/src/appui-react/statusbar/Popup.tsx

```diff
@@ -14,5 +14,5 @@ import { FooterPopup } from "../layout/footer/Popup";
  * @beta
  */
 export function StatusBarPopup(props: Partial<PopupProps>) {
-  return <FooterPopup {...props} />;
+  return <FooterPopup showArrow={false} {...props} />;
```

Comment:
Ouch, `showArrow` is a public prop in `PopupProps` and used by `StatusBarPopup`...
I think we want to disallow that configurability aspect? I.e. currently someone might still add arrow to the `StatusBarPopup` since the props are spread after the `showArrow`.

Reply:
Switched the order.